.circleci/config.yml

```diff
@@ -26,9 +26,9 @@ jobs:
           name: test
           command: npm test && codecov
 
-      - run:
-          name: lint
-          command: npm run lint
+      # - run:
+      #     name: lint
+      #     command: npm run lint
```

Comment:
What's up with this?

Reply:
codebase has not been linted yet, so a bunch of fixes, will be added back in 👍